service/controller/drainer.go

```diff
@@ -44,10 +49,10 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 			NewRuntimeObjectFunc: func() runtime.Object {
 				return new(corev1.Pod)
 			},
-			Logger:       config.Logger,
-			ResourceSets: resourceSets,
+			Logger:    config.Logger,
+			Resources: resources,
 			Selector: labels.SelectorFromSet(map[string]string{
-				key.PodWatcherLabel: project.Name(),
```

Comment:
I'm not totally confident about this change. I will test draining. Comments welcome.

Reply:
Yes this seems incorrect. It should still be project.Name IMO. This chantge here could have unintended consequences for updates where the version might not be the project version version (while upgrading the nodes).

Reply:
I felt like it should be version specific otherwise all operators will watch all pods. I will revert it though if it seems like a bad idea--it evidently didn't cause problems in the past.